db/migrate/20170912202756_add_locale_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :locale, :string
```

Comment:
Can we add a default here? since we have old users, will be great to have those users in consideration